cmd/dendritejs/main.go

```diff
@@ -69,10 +68,10 @@ func createFederationClient(cfg *config.Dendrite, node *go_http_js_libp2p.P2pLoc
 	return fed
 }
 
-func createP2PNode() (serverName string, node *go_http_js_libp2p.P2pLocalNode) {
-	hosted := "/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star"
-	_ = "/ip4/127.0.0.1/tcp/9090/ws/p2p-websocket-star/"
-	node = go_http_js_libp2p.NewP2pLocalNode("org.matrix.p2p.experiment", []string{hosted})
+func createP2PNode(privKey ed25519.PrivateKey) (serverName string, node *go_http_js_libp2p.P2pLocalNode) {
+	_ = "/dns4/ws-star.discovery.libp2p.io/tcp/443/wss/p2p-websocket-star"
+	hosted := "/ip4/127.0.0.1/tcp/9090/ws/p2p-websocket-star/"
```

Comment:
Should this be a public rendezvous for now or is the plan to wait for a Matrix rendezvous? If not then might be worth dropping this requirement into the P2P readme.

Reply:
It already is in the README - https://github.com/matrix-org/dendrite/blob/master/p2p.md#rendezvous - this will be configurable in the future.